go.mod

```diff
@@ -129,11 +129,12 @@ require (
 	go.etcd.io/etcd v3.3.11+incompatible
 	go.mongodb.org/mongo-driver v1.0.0
 	go.opencensus.io v0.19.2
-	golang.org/x/crypto v0.0.0-20190325154230-a5d413f7728c
-	golang.org/x/net v0.0.0-20190328230028-74de082e2cca
```

Comment:
Why these changes?

Reply:
This should probably be its own module since it's really not part of the CDK but a standalone tool.

Reply:
Created a new module. Opened #1883 to document a near-miss I had while doing it.